core/src/components/button/button.ios.vars.scss

```diff
@@ -151,13 +151,13 @@ $button-ios-outline-background-color-focused:         ion-color(primary, base, $
 // --------------------------------------------------
 
 /// @prop - Font size of the clear button
-$button-ios-clear-font-size:                          17px !default;
+$button-ios-clear-font-size:                          min(1.0625rem, 51px) !default;
```

Comment:
Same as with the solid, the actual value of the `font-size` at `310%` is closer to `52px` but I thought 3x would be nicer.